modules/serverless-install-eventing-yaml.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+//  * serverless/installing_serverless/installing-knative-eventing.adoc
+
+[id="serverless-install-eventing-yaml_{context}"]
+= Installing Knative Eventing using YAML
+
+.Procedure
+
+. Create a file named `eventing.yaml`.
```

Comment:
The web console version is much more descriptive. I think we should post here the same information, about customizing the YAML file. Also, it would be wise to put here, the `spec` subkey with default configuration, so that users will know what options they have.

BTW. Those knative eventing & serving options should be documented somewhere. That's probably can be tracked as separate issue.

Reply:
Can you please raise an issue for this @cardil with the details you'd like me to include, i.e. `spec` default config example?

Reply:
I created [SRVKS-552](https://issues.redhat.com/browse/SRVKS-552) and [SRVKE-407](https://issues.redhat.com/browse/SRVKE-407)